build.gradle.kts

```diff
@@ -3,6 +3,7 @@ plugins {
     id("org.springframework.boot") version "2.5.6"
     id("org.jlleitschuh.gradle.ktlint") version "10.1.0"
     kotlin("jvm") version "1.5.31"
+    kotlin("plugin.allopen") version "1.4.32"
```

Comment:
Kotlin version is wrong, should be 1.5.31

Reply:
kotlin version changed in commit e276a4b